### ISSUE TITLE
Moderate Accessibility Violation scope="row" outside table header

### ISSUE THREAD

**Issue by terrazoon** [GSA/notifications-admin#2001]
tests/end_to_end/test_send_message_from_existing_template.py::test_send_message_from_existing_template[firefox] - AssertionError: Accessibility violation: {'id': 'scope-attr-valid', 'impact': 'moderate', 'tags': ['cat.tables', 'best-practice'], 'description': 'Ensures the scope attribute is used correctly on tables', 'help': 'scope attribute should be used correctly', 'helpUrl': 'https://dequeuniversity.com/rules/axe/4.4/scope-attr-valid?application=axeAPI', 'nodes': [{'any': [], 'all': [{'id': 'html5-scope', 'data': None, 'relatedNodes': [], 'impact': 'moderate', 'message': 'In HTML 5, scope attributes may only be used on table header elements (<th>)'}], 'none': [], 'impact': 'moderate', 'html': '<td class="table-field jobid" scope="row" role="rowheader">\n                  <a class="usa-link" href="/services/e4bd74be-2a94-4138-aa28-a2b2da2d1ac6/jobs/aecfcbc2-503b-4d0c-b96c-df20ef085446">\n                    aecfcbc2\n                  </a>\n                </td>', 'target': ['.jobid'], 'failureSummary': 'Fix all of the following:\n  In HTML 5, scope attributes may only be used on table header elements (<th>)'}]}
assert 'moderate' in ['minor']

**Comment by terrazoon** [comment]
To see the error, edit the /tests/end_to_end/conftest.py file to not skip 'moderate' violations and then run the e2e tests.
